zh-hans/lessons/ecto/changesets.md

```diff
@@ -213,7 +213,7 @@ end
 iex> User.changeset(%User{}, %{"name" => "Bob"})
 #Ecto.Changeset<
   action: nil,
-  changes: %{first_name: "Bob"},
+  changes: %{name: "Bob"},
```

Comment:
Care to do another PR applying this change to all languages impacted?

Reply:
Will work on this after work tonight.